lib/rust/parser/src/syntax/item.rs

```diff
@@ -73,6 +83,34 @@ impl<'s> TryAsRef<Item<'s>> for Item<'s> {
     }
 }
 
+/// Given a sequence of `Item`s belonging to one block, create an AST block node, of a type
```

Comment:
You can refer to ``` [`Item`] ``` in the docs - we try to use this syntax whenever its possible to refer to exisitng entities.